pandas/tests/series/methods/test_drop_duplicates.py

```diff
@@ -67,76 +67,109 @@ def test_drop_duplicates_no_duplicates(any_numpy_dtype, keep, values):
 
 
 class TestSeriesDropDuplicates:
-    @pytest.mark.parametrize(
-        "dtype",
-        ["int_", "uint", "float_", "unicode_", "timedelta64[h]", "datetime64[D]"],
+    @pytest.fixture(
```

Comment:
maybe separate

but can add EA types generally to dtype lists like this

Reply:
yah, for now just doing the splitting+fixturizing that made it into #36131 today